frontend/src/components/Terminal.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import '../../node_modules/@wasmer/wasm-terminal/lib/xterm/xterm.css';
```

Comment:
You shouldn't have to do this -- webpack should be set up so that you can just do `import @wasmer/wasm-terminal/lib/xterm/xterm.css`